<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Idas-MacBook-Air.local" name="se.nrm.dina.data.exceptions.DinaExceptionNGTest" tests="4" failures="0" timestamp="20 Apr 2016 11:58:10 GMT" time="0.114" errors="0">
  <testcase name="testDinaExceptionConstructWithMsgAndErrorCode" time="0.001" classname="se.nrm.dina.data.exceptions.DinaExceptionNGTest"/>
  <testcase name="testGetErrorCode" time="0.001" classname="se.nrm.dina.data.exceptions.DinaExceptionNGTest"/>
  <testcase name="testDinaExceptionConstruct" time="0.099" classname="se.nrm.dina.data.exceptions.DinaExceptionNGTest"/>
  <testcase name="testDinaExceptionConstructWithMsg" time="0.013" classname="se.nrm.dina.data.exceptions.DinaExceptionNGTest"/>
</testsuite> <!-- se.nrm.dina.data.exceptions.DinaExceptionNGTest -->
